test/ClientTest.php

```diff
@@ -524,4 +524,28 @@ public function testFormUrlEncodeSeparator()
         $rawRequest = $client->getLastRawRequest();
         $this->assertContains('foo=bar&baz=foo', $rawRequest);
     }
+
+    public function uriDataProvider()
+    {
+        return [
+            ['/example', true],
+            ['http://localhost/example', false]
+        ];
+    }
+
+    /**
+     * @dataProvider uriDataProvider
+     */
+    public function testValidRelativeURI($uri, $isValidRelativeURI)
+    {
+        $client = new Client($uri);
+        $this->assertSame($isValidRelativeURI, $client->getUri()->isValidRelative());
+    }
+
+    public function testSendRequestWithRelativeURI()
+    {
+        $client = new Client('/example');
+        $client->setAdapter(Test::class);
+        $client->send();
```

Comment:
There is no any assertion in that test. It is really unclear what we are testing here.

Reply:
done ;)